documentation/user-interface/test-editor.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+title: Test Editor
+permalink: /documentation/user-interface/test-editor.html
+---
+# Test Editor
+
+The **Test** editor allows you to view and modify existing tests, and create new ones. The editor opens when you double-click the test in the [Explorer](explorer-panel.md) panel or click it in the [Fixture](fixture-editor.md) editor. If you [create an empty test](../working-with-testcafe-studio/recording-tests/README.md#creating-tests) or start [recording a new test](../working-with-testcafe-studio/recording-tests/README.md#starting-and-stopping-recording), TestCafe Studio automatically opens the test in the editor.
```

Comment:
> The editor opens when you double-click the test in the Explorer panel or click it in the Fixture editor.

I don't like how `it` here refers to `the test`. After all, you click different things despite that they represent a single test. Maybe it's better to repeat? 

```
The editor opens when you double-click a test in the Explorer panel or click a test in the Fixture editor.
```